examples/react-es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "react": "0.13.0-beta.1",
+    "react-tools": "0.13.0-beta.1"
+  },
+  "devDependencies": {
+    "babel-jest": "*"
```

Comment:
Hmm, are you sure? If I just use that I get this error

```bash
emmenko: ~/dev/src/jest/examples/react-es6 (react-es6 *%) $ npm test

> @ test /Users/emmenko/dev/src/jest/examples/react-es6
> node ../../bin/jest.js

Using Jest CLI v0.2.2
 FAIL  __tests__/CheckboxWithLabel-test.js
Error: /Users/emmenko/dev/src/jest/examples/react-es6/preprocessor.js: Parse Error: Line 1: Illegal import declaration
1 test failed, 0 tests passed (1 total)
```

with this `preprocessor`

```javascript
var ReactTools = require('react-tools');

module.exports = {
  process: function(src) {
    return ReactTools.transform(src, {harmony: true});
  }
};
```

...or am I missing something?